index.js

```diff
@@ -42,6 +46,7 @@ module.exports = {
     "max-line-length": [ 80, {
       ignore: "non-comments",
     } ],
+    "max-nesting-depth": 3,
```

Comment:
Is this arbitrary, or is three a good recommendation from somewhere? SUIT hasn't really considered nesting before but we're starting to think about it in the bem-linter as you saw. Happy to decide on some rules here.